scripts/combine/setupCombine.py

```diff
@@ -242,15 +268,18 @@ def setup(args, inputFile, fitvar, xnorm=False):
         
     passSystToFakes = wmass and not args.skipSignalSystOnFakes and args.qcdProcessName not in excludeGroup and (filterGroup == None or args.qcdProcessName in filterGroup) and not xnorm
 
-    cardTool.addProcessGroup("single_v_samples", lambda x: x[0] in ["W", "Z"] and x[1] not in ["W","Z"])
+    # FIXME: try to reuse some lambda functions to avoid mistakes, or maybe define some general functions?
+    cardTool.addProcessGroup("single_v_samples", lambda x: any(x.startswith(init) for init in ["W", "Z", "BkgW", "BkgZ"]) and x[1] not in ["W","Z"])
```

Comment:
Maybe W_OOA is better than Bkg? Or is it needed? I think this should also be a function stored somewhere else (is_w_z_sample or similar) that is called both here and in the histmaker, since now we have both single_v_procs and this check in different places

Reply:
I was initially undecided between OutOfAccWmunu and BkgWmunu, the unfolding was using BkgWmunu so I kept it. One thing to be careful is whether the name has to start with W or not (as I did it it does not), because there are many conditions around the code set for signal based on how the process name starts with W (or Z for Wlike/dilepton). Technically we want to include the OOA there, but it is sometimes simpler to add it explicitly when it is present, rather than applying conditions to exclude it.
Perhaps this can be sorted out later once we get a better understanding of how we want to use this process (and then I can make it more homogeneous with the unfolding too).